ci-automation/garbage_collect.sh

```diff
@@ -256,6 +256,7 @@ function _garbage_collect_impl() {
     docker run --pull always --rm --net host \
       --env AZURE_AUTH_CREDENTIALS --env AZURE_PROFILE \
       --env AWS_ACCESS_KEY_ID --env AWS_SECRET_ACCESS_KEY \
+      --env AWS_CREDENTIALS \
```

Comment:
I suppose that these are yet different stuff from `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`?